### DISCUSSION TITLE
Choosing appropriate AST-GAS component

### DISCUSSION THREAD

**Discussion by hsaglik** [firemodels/fds#13861] [Q&A] [answered]
Hi.
I have a pool fire model and I want to get “Adiabatic Surface Temperature Gas” from a device near the fire to further use in following detailed heat transfer in finite element models, such as in Abaqus. I cannot use the AST-Solid device because I need approximate ASTs without solid obstruction in FDS.
The User Guide defines the AST using incident radiative heat flux and gas temperature around the device. I determined the emissivity and heat transfer coefficients. However, an ORIENTATION vector should be defined. At the same point, I defined 6 AST-GAS devices whose orientation vectors are selected in 6 directions to compare the magnitudes: -x, +x, -y, +y, -z, and +z. Also, I defined a TEMPERATURE device at the same location to see the GAS temperature.
My question is which component of AST-GAS I should use in the following analysis. Because, in reality, a single point on a surface near to fire is exposed to radiation from at least 5 directions – the remaining 1 direction would not have a view factor from the fire. 
The average ASTs found in my model during steady-state with respect to 6 directions are given below. As seen, the AST values in the directions that are not directly toward the fire are also considerable. I assume the total incident radiative heat flux on the device is divided into direction components and ASTs in 6 directions are found by using gas temperature and heat transfer coefficients.
By adopting AST from one direction, say the direction with the larger AST value, wouldn't I still be below the true AST temperature? If so, is there any method to get one AST value that uses the combined effect of incident radiative heat flux on the device?

![Weixin Image_20241209135544](https://github.com/user-attachments/assets/a65cab5e-940e-4a7a-af21-5d48bbac3c36)

**Comment by drjfloyd** [comment, marked as answer]
The ORIENTATION for the gas phase AST does not mean the DEVC sees radiation only from that angle. That ORIENTATION represents the surface normal. The incident radiation is then integrated over the 2 pi angles around that normal using the cosine of the angle of the incident radiaiton with the surface normal.

**Reply by hsaglik** [reply]
Oh, thank you so much for the reply! I feel relieved.
